.eslintrc.js

```diff
@@ -78,4 +78,7 @@ module.exports = {
       },
     },
   },
+  parserOptions: {
+    project: ["tsconfig.json","examples/example-react-vite/tsconfig.json"]
+  }
```

Comment:
Was this needed for a specific error/issue?

Reply:
Yes, IntelliJ shows an error in every *.ts, *.tsx file when this isn't set - it was annoying. :)